integration/operator_test.go

```diff
@@ -82,3 +82,46 @@ func TestGetReleaseVersion(t *testing.T) {
 		t.Errorf("release didn't match expected, want %q, got %q", expected, actual)
 	}
 }
+
+func TestDeleteRelease(t *testing.T) {
```

Comment:
I would like to split integration tests from the beginning and not pile up all of them together. We can then run them in parallel and rebuild separate tests in case we want to. Also the project will be more clean in general.

Reply:
IMO we can think of running in parallel when needed, I think it's much clearer having all of them in a single file. Also, the build is now just 2min, blocking circleCI machine slots for no reason doesn't make too much sense.

Reply:
>  I think it's much clearer having all of them in a single file.

No it is not and I have a very hard time fighting through certain projects that have everything in the same file. It would be nice to have some separation of concerns and some decent design so projects are structured properly and can be maintained more easily at a later point. I am doing so much yak shaving currently and I would like to reduce this in the future.